contents/tutorials/api-get-insights-persons.mdx

```diff
@@ -15,9 +20,14 @@ This tutorial shows you the basics of the PostHog API. We focus on `GET` request
 
 Like most APIs, we must authenticate ourselves before we can access our data in PostHog. To do this, you need a personal API key. 
 
-To create a personal API key in PostHog, click your avatar icon in the bottom left corner of your PostHog instance, then click the gear icon to go to "My settings." 
+To create a personal API key in PostHog, click your avatar icon in the bottom left corner of your PostHog instance, then click the gear icon to go to "Account settings." 
 
-![Profile](../images/tutorials/api-get-insights-persons/profile.png)
+<ProductScreenshot
+    imageLight = {profileLight} 
+    imageDark = {profileDark}
+    classes="rounded"
+    alt="Profile"
+/>
```

Comment:
@jasmineyas heads up your gmail email address is publicly viewable here.

Reply:
thanks for noting that - blurred!